### DISCUSSION TITLE
How to create 3D model without AR using arsceneview

### DISCUSSION THREAD

**Discussion by rsrakesh3** [sceneview/sceneview#246] [General]
Is it possible to use arsceneview for both ar and 3d model(without camera).

**Comment by grassydragon** [comment]
Hi!
I think it can be easier to create two separate screens for 3D and AR.

**Reply by rsrakesh3** [reply]
I mean is it possible to create AR as well as 3D view using one library either arsceneview or sceneview. As including both the dependencies are causing conflicts.
@grassydragon @hshapley @StevenMohr @paul678

**Reply by grassydragon** [reply]
Yes, you can include only `arsceneview`.

**Reply by rsrakesh3** [reply]
Thank you @grassydragon Is there any code snippet to achieve 3D view using **arsceneview**

**Reply by grassydragon** [reply]
You should be able to follow an older version of the `model-viewer` sample (before the components were introduced in `sceneview`): https://github.com/SceneView/sceneview-android/tree/199b35609cf62a286bdcf25eaa2fe76324eabced/samples/model-viewer/src/main/java/io/github/sceneview/sample/modelviewer

**Comment by ThomasGorisse** [comment]
In v4.0.0-rc.1, use the `sceneview` artifact (not `arsceneview`) for 3D without AR:

```kotlin
implementation("io.github.sceneview:sceneview:4.0.0-rc.1")

@Composable
fun My3DScene() {
    SceneView {
        val model = rememberModelInstance(modelLoader, "models/mymodel.glb")
        if (model != null) {
            ModelNode(modelInstance = model, scaleToUnits = 1.0f)
        }
    }
}
```

`SceneView` = 3D only (no camera, no ARCore). `ARSceneView` = 3D + camera + ARCore. They are separate artifacts so you don't need to include ARCore dependencies for a simple 3D viewer.
